readme.md

```diff
@@ -100,6 +100,11 @@ services:
       - <path/to/config>:/config
 ```
 
+### Updating Docker Compose
+```bash
+docker  docker-compose -f <path/to/config>/docker-compose.yml pull 
```

Comment:
looks like you have an extra space here

Reply:
A couple things of note / suggestion:
- for readability (as this is the readme) use `--file` instead of `f`
- remove the leading `docker`
- for compose v2 the command will be `docker compose --file <path/to/config>/docker-compose.yml pull`
- for compose v1 the command will be `docker-compose --file <path/to/config>/docker-compose.yml pull`